content/en/hub/_index.md

```diff
@@ -12,10 +12,10 @@ no_list: true
 	</tr>
 </table>
 
-<p style="max-width:100%;">TrueNAS® is an operating system that is designed to be the most effective and time-efficient solution for storing, securing, and sharing data over a network.
-	It is the simplest way to create a safe, centralized, and easily accessible place for your data.<br><br>
-	The topics on this page are arranged to guide you through learning about TrueNAS, installing and configuring the system, then setting up data storage and sharing.
-	There are additional topics for the TrueCommand and TrueNAS SCALE members of the TrueNAS family, 3rd party software integration notes, and reference documentation for the powerful and cost-effective TrueNAS storage appliances available for purchase from iXsystems.
+<p style="max-width:100%;">TrueNAS® is an embedded open source network-attached storage (NAS) operating system that is designed to be the most effective and time-efficient solution for storing, securing, and sharing data over a network.
+It is released under a <a href="https://opensource.org/licenses/BSD-2-Clause" target="_blank">2-clause BSD license</a> and is the simplest way to create a safe, centralized, and easily accessible place for your data.<br><br>
```

Comment:
Since parts of the TrueNAS OS are under GPL... do we need to state that?  All of our own custom work is BSD, FreeBSD Base is BSD, but ports we utilize are GPL.
Not sure if saying the whole thing is BSD could be an problem for us legally, if one of the GPL projects we rely on decides to take issue with that claim.